zuul.d/jobs.yaml

```
@@ -719,99 +780,126 @@
       - huaweicloud_credentials
     override-checkout: v1.3.3
 
-- job:
-    name: packer-master-functional-devstack
-    parent: init-test
-    description: |
-      Test image building functionality of Packer of master against devstack.
-    run: playbooks/packer-functional-devstack/run.yaml
-
 - job:
     name: packer-functional-devstack
     parent: init-test
     description: |
-      Test image building functionality of Packer of v1.3.3 against master devstack.
+      Test image building functionality of master Packer against devstack.
```

```suggestion
      Test image building functionality of master Packer against master OpenStack.
```
Same issues in following.

---

done